Datadog.xcworkspace/contents.xcworkspacedata

```diff
@@ -7,7 +7,4 @@
    <FileRef
       location = "group:instrumented-tests/http-server-mock">
    </FileRef>
-   <FileRef
-      location = "group:session-replay">
-   </FileRef>
```

Comment:
I had to disintegrate `session-replay` module from the main workspace and `Example` app target. This is because Xcode fails to resolve the dependencies within the workspace:

<img width="244" alt="Screenshot 2022-10-26 at 14 08 13" src="https://user-images.githubusercontent.com/2358722/198022845-1485d1cb-2cf7-4f26-9f65-cb7aff11042d.png">

<img width="379" alt="Screenshot 2022-10-26 at 14 07 59" src="https://user-images.githubusercontent.com/2358722/198023283-d2b29c1d-f4e3-4599-8d93-cbca2e1ea588.png">

I think it's because `Datadog.pbxproj` already defines `Datadog*` modules and the same modules are shipped through local dependency on `dd-sdk-ios` package. I considedr it a "workspace" factor, because the same setup (depending on local `session-replay` package that links local `dd-sdk-ios`) works in other local projects I have.

Losing presence of `session-replay` in `Example` app's workspace is not a problem for me, as I rarely test SR in this app (I use better ones - with more challenging UIs). Also, solving this problem could be difficult if not impossible without the new project structure (RFC is in progress).